src/cpu/instruction.rs

```diff
@@ -652,6 +660,46 @@ pub enum CbInstruction {
 impl CbInstruction {
     pub fn from_byte(byte: u8) -> Option<CbInstruction> {
         match byte {
+            // rlc register 8 bit
```

Comment:
stick with the naming convention from the manual for these comments

Reply:
You want like, "rlc n"?